web/scripts/frame.js

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I noticed many of our checked-in JS files do not have a copyright at the top. Should they?

Reply:
Yeah I'd be happy to. We only have a handful of JS files. @RedBrogdon do you know?

Reply:
I'll make this change in a follow-up.